### DISCUSSION TITLE
Search Menu Display

### DISCUSSION THREAD

**Discussion by kaclark** [pystardust/ytfzf#680] [Q&A]
Hello,

I am simply dazzled by the way that the menu looks like when search results are returned. I was wondering what the minimal code needed to reproduce this effect for other projects would look like. Would it be possible for this to be provided, or can I be pointed in a proper direction for figuring out something similar?

Thank you the awesome work. Truly impressive.

**Comment by Euro20179** [comment]
The menu simply uses fzf with the --preview option.
